src/main/java/frc5124/robot2020/commands/turret/returnTurretToStart.java

```diff
@@ -0,0 +1,67 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc5124.robot2020.commands.turret;
+
+import com.revrobotics.CANPIDController;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc5124.robot2020.subsystems.Turret;
+
+public class returnTurretToStart extends CommandBase {
+  private Turret turret;
+  private CANPIDController turretPID;
+  private boolean isDone = false;
+
+  /**
+   * Creates a new returnTurretToStart.
+   */
+  public returnTurretToStart(Turret subsystem) {
+    turret = subsystem;
+    addRequirements(turret);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    turretPID = turret.getMotor().getPIDController();
```

Comment:
PID controller should be constructed and configured in the subsystem not in the command